component/otelcol/processor/transform/transform.go

```diff
@@ -53,9 +53,9 @@ func (c *ContextID) UnmarshalText(text []byte) error {
 	}
 }
 
-type contextStatementsSlice []contextStatements
+type contextStatementsSlice []ContextStatements
```

Comment:
IMO this type should be exported too; I think it's a mistake that it's not already.